client/src/resources/roles.json

```diff
@@ -5,263 +5,487 @@
+        "chatMessages":[
+            {"type": "veteranAttackedVisitor"},
+            {"type": "veteranAttackedYou"},
+            {"type": "someoneSurvivedYourAttack"}
+        ]
     },
     "deputy": {
         "faction": "town",
         "armor": false,
         "maxCount": 1,
         "largeRoleSpecificMenu": false,
         "keywords": ["defense","roleLimit"],
-        "canBeConvertedTo": []
+        "canBeConvertedTo": [],
+        "chatMessages":[
+            {"type": "deputyKilled", "shotIndex": 11},
+            {"type": "deputyShotYou"},
+            {"type": "someoneSurvivedYourAttack"}
+        ]
     },
     "escort": {
         "faction": "town",
         "armor": false,
         "maxCount": null,
         "largeRoleSpecificMenu": false,
         "keywords": ["roleblock"],
-        "canBeConvertedTo": []
+        "canBeConvertedTo": [],
+        "chatMessages":[
+            {"type": "roleBlocked", "immune": true},
+            {"type": "roleBlocked", "immune": false}
+        ]
     },
     "medium": {
         "faction": "town",
         "armor": false,
         "maxCount": null,
         "largeRoleSpecificMenu": false,
         "keywords": ["haunt","dead","chatGroup"],
-        "canBeConvertedTo": []
+        "canBeConvertedTo": [],
+        "chatMessages":[
+            {
+                "type": "normal", 
+                "messageSender": {
+                    "type": "medium"
+                },
+                "text": "Anyone wanna throw me around like a football?", 
+                "chatGroup": "dead"
+            },
+            {
+                "type": "normal", 
+                "messageSender": {
+                    "type": "player",
+                    "player": 37
+                },
+                "text": "I wanna throw you around like a football", 
+                "chatGroup": "dead"
+            },
+            {"type": "mediumSeance", "player": 10}
+        ]
     },
     "mayor":{
         "faction": "town",
         "armor": false,
         "maxCount": 1,
         "largeRoleSpecificMenu": false,
         "keywords": ["roleLabel","roleLimit"],
-        "canBeConvertedTo": []
+        "canBeConvertedTo": [],
+        "chatMessages":[
+            {"type": "mayorRevealed","playerIndex": 3}
+        ]
     },
     "journalist": {
         "faction": "town",
         "armor": false,
         "maxCount": 1,
         "largeRoleSpecificMenu": true,
         "keywords": ["interview","journal","chatGroup","roleLimit"],
-        "canBeConvertedTo": []
+        "canBeConvertedTo": [],
+        "chatMessages":[
+            {"type": "youAreInterviewingPlayer","playerIndex": 1},
+            {"type": "playerIsBeingInterviewed","playerIndex": 1},
+            {
+                "type": "normal", 
+                "messageSender": {
+                    "type": "journalist"
+                },
+                "text": "Could you give us your opinion on the situation happening in [REDACTED]?", 
```

Comment:
This is funny, but people might think [REDACTED] has some special meaning.
Consider:
![image](https://github.com/mafia-rust/mafia/assets/64770632/d2565444-a50d-467c-a30f-7aa344993fc3)
Making medium this ^ and journalist the football one